packages/web-components/src/components/table-of-contents/table-of-contents.ts

```diff
@@ -791,20 +805,32 @@ class C4DTableOfContents extends MediaQueryMixin(
             : ``}
           <div
             class="${c4dPrefix}-ce--table-of-contents__items-container"
+            part="container"
```

Comment:
We already have a `part="container"` for something else (line 766).
```suggestion
            part="items-container"
```